model/user.py

```diff
@@ -1 +1,62 @@
 """Data model to represent an individual user."""
+from permissions import Permissions
```

Comment:
Looks good to me. Can you write unit tests for all these methods? (I forgot to include it in the original ticket. 😞) You can probably follow Robin's [teams tests](https://github.com/ubclaunchpad/rocket2.0/blob/master/tests/model/team_test.py) pretty closely.

Reply:
Oh, also, do keep in mind that any tests you add will also need docstrings.